.gitignore

```diff
@@ -0,0 +1,2 @@
+# This file should be ignored by git
+test.md
```

Comment:
- [ ] Please kindly add the `node_modules` to your `.gitignore file` ☝️ ☝️

Reply:
I have made the necessary changes required in the project repo.